ui/apps/platform/src/Containers/Integrations/IntegrationForm/Forms/AzureIntegrationForm.tsx

```diff
@@ -271,4 +271,4 @@ function ClairifyIntegrationForm({
     );
 }
 
-export default ClairifyIntegrationForm;
```

Comment:
Good eyes! To answer your question,
* For `export default whatever` the local name and the import name can differ.
    Because `export default` is community convention for React components, we follow the pattern.
    A more subtle occasional inconsistency is when file name differs from component name.
* For `export const whatever` or `export function whatever` the name must match.
    Some authors of command line JavaScript tools use named export and import only.